dashboard/modules/snapshot/snapshot_schema.json

```diff
@@ -120,6 +120,21 @@
                   "port": {
                     "type": "integer"
                   },
+                  "serveMetadata": {
```

Comment:
So our we going with this metadata approach on the actor? I prefer this tbh. But I thought this wasn't feasible

Reply:
Yeah we decided we shouldn't modify the actor table in the GCS, but it turns out it's simple to add in this data on the layer above, just in the snapshot itself.